src/lib/poll/poll.ts

```diff
@@ -41,13 +41,13 @@ export const poll = <T>({
   const execute = async (resolve: (arg: T) => void, reject: (err: unknown) => void) => {
     for (let attempts = 0; attempts < maxAttempts; attempts++) {
       if (isCancelled) return // dont resolve/reject - leave promise on event loop
+      await sleep(interval)
```

Comment:
Moved to the beginning of the method so we honor the interval on the first call

Reply:
🧠

Reply:
does this mean we get 
```
sleep -> request -> sleep -> request -> ...
```
or
```
request -> sleep -> request -> sleep -> ...
```
?

Because the former will introduce `interval` millis of latency to each initial call

Reply:
https://github.com/shapeshift/web/pull/5640/commits/42c6ad6700640069942d92f6f32705a004c7f364